ui/src/mixins/FocusTrap.ts

```diff
@@ -0,0 +1,23 @@
+import Vue from "vue";
+import createFocusTrap, { FocusTrap, Options } from "focus-trap";
+
+let focusTrap: FocusTrap;
```

Comment:
By having a single global `let focusTrap`-variable we are basically saying that there can only ever be a *one* focus trap active at a given time. Right? What if, further down the road, the focus trap mixin is also used in, let's say the Popover-component (which is used by the combobox-component) and the app has a combobox inside a modal?

My intuition says that the mixin needs non-reactive state:

```js
{
  created() {
    this.focusTrap = null; // maybe we should prefix this with fd?
  }
}
```

What do you think?

Reply:
Yes, that's true. I will change it accordingly

Reply:
I fixed it in the latest commit